sbin/autograder/grade_item_main_runner.py

```diff
@@ -0,0 +1,79 @@
+import configparser
+import json
+import os
+import tempfile
+import shutil
+import subprocess
+import stat
+import time
+import dateutil
+import dateutil.parser
+import urllib.parse
+import string
+import random
+import socket
+import zipfile
+import traceback
+
+from submitty_utils import dateutils, glob
+from . import grade_items_logging, write_grade_history, CONFIG_PATH
+
+with open(os.path.join(CONFIG_PATH, 'submitty.json')) as open_file:
+    OPEN_JSON = json.load(open_file)
+SUBMITTY_INSTALL_DIR = OPEN_JSON['submitty_install_dir']
+SUBMITTY_DATA_DIR = OPEN_JSON['submitty_data_dir']
+
+with open(os.path.join(CONFIG_PATH, 'submitty_users.json')) as open_file:
+    OPEN_JSON = json.load(open_file)
+DAEMON_UID = OPEN_JSON['daemon_uid']
+
+
+def executeTestcases(complete_config_obj, tmp_logs, tmp_work, queue_obj, submission_string, item_name, USE_DOCKER, container, which_untrusted):
```

Comment:
Talked about this briefly yesterday, but there's no real point in breaking this function out into its own module and should probably just be left in grade_item.py which is a module centered around grading an item (which this function is a part of doing).